src/VirtualClient/VirtualClient.Main/profiles/PERF-APACHEBENCH.json

```diff
@@ -0,0 +1,50 @@
+{
+  "Description": "Apache HTTP Server Benchmarking Workload",
+  "MinimumExecutionInterval": "00:01:00",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "01:00:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64,win-x64,win-arm64",
+    "SupportedOperatingSystems": "Debian,Ubuntu,Windows"
+  },
+  "Actions": [
+    {
+      "Type": "ApacheBenchExecutor",
+      "Parameters": {
+        "Scenario": "ApacheBench_N50000_C10",
+        "SupportedPlatforms": "win-x64, win-arm64, linux-x64, linux-arm64",
+        "PackageName": "apachehttpserver",
+        "CommandLine": "-k -n 50000 -c 10 http://localhost:80/"
```

Comment:
I think this should come as a a parameter so that user can amend it from outside